types/stripe/index.d.ts

```diff
@@ -4979,6 +5021,59 @@ declare namespace Stripe {
         }
     }
 
+    namespace reviews {
+        interface IReview extends IResourceObject {
+            /**
+             * Unique identifier for the object.
+             */
+            "id": string;
+
+            /**
+             * String representing the object’s type. Objects of the same type share the same value.
+             */
+            "object": "review";
+            
+            /**
+             * The charge associated with this review. [Expandable]
+             */
+            "charge": string | charges.ICharge;
+
+            /**
+             * Time at which the object was created. Measured in seconds since the Unix epoch.
+             */
+            "created": number;
+
+            /**
+             * Has the value true if the object exists in live mode or the value false if the object exists in test mode.
+             */
+            "livemode": false;
+
+            /**
+             * If true, the review needs action.
+             */
+            "open": true;
+
+            /**
+             * The reason the review is currently open or closed.
+             */
+            "reason": "rule" | "manual" | "approved" | "refunded" | "refunded_as_fraud" | "disputed";
+        }
+    }
+
+    namespace applications {
+        interface IApplication extends IResourceObject {
```

Comment:
Have you got any documentation for where you got these properties? I can't seem to find any application object documented on the Stripe API docs.

Reply:
I see why you've done this though, as the `application` property is `Expandable`

Reply:
Yes. Actually I figured this out from the .Net API docs.